docs/source/trainer/disributed_training.rst

```diff
@@ -0,0 +1,29 @@
+Distributed Training
```

Comment:
This is copied from our current getting started section (from which it is removed)